desk-validation.js

```diff
@@ -0,0 +1,53 @@
+const deskEmail = document.querySelector('#desk-email');
+const deskForm = document.querySelector('#desk-form');
+const isRequired = (value) => (value !== '');
+
+const isEmailValid = (email) => {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@(([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(email);
+};
+
+const showError = (input, message) => {
+  // get the form-field element
+  const formField = input.parentElement;
+  // add the error class
+  formField.classList.remove('success');
+  formField.classList.add('error');
+
+  // show the error message
+  const error = formField.querySelector('small');
+  error.textContent = message;
+};
+
```

Comment:
**[REQUIRED CHANGES]**

- Hey, while you made a lot of effort in implementing the changes required by the previous review, there's still an issue with how you've displayed the error message acccording to [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md). Please, see the image below:

![instruction](https://user-images.githubusercontent.com/59397159/169556594-3d14aef7-58d6-42ac-8329-a0de4bd03fe2.png)

> The error message should appear near or under the submit button. please kindly fix this.